exporter.go

```diff
@@ -383,14 +383,14 @@ func main() {
 		registry.MustRegister(collector.NewNginxCollector(ossClient.(*client.NginxClient), "nginx", constLabels.labels))
 	}
 	http.Handle(*metricsPath, promhttp.HandlerFor(registry, promhttp.HandlerOpts{}))
+
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		_, err := w.Write([]byte(`<html>
-			<head><title>NGINX Exporter</title></head>
-			<body>
+		_, err := fmt.Fprintf(w, `<!DOCTYPE html>
+			<meta charset="utf-8">
```

Comment:
not sure if meta is neccesary, because the exporter already includes the charset in the response header:
```
$ curl http://localhost:9113/ -I
HTTP/1.1 200 OK
. . .
Content-Type: text/html; charset=utf-8
```

Reply:
Right, I've removed it.